src/app.py

```diff
@@ -0,0 +1,36 @@
+from quart import Quart, request
+from openai import AzureOpenAI
+from dotenv import load_dotenv
+import os
```

Comment:
Please use isort to sort imports (os should be first).

Reply:
Sorted. Thanks for the recommendation.